actions/feedback_actions.py

```diff
@@ -126,13 +125,17 @@ def extract_feedback_type(
         dispatcher: CollectingDispatcher, tracker: Tracker, domain: DomainDict
     ) -> Dict[Text, Any]:
         if tracker.get_slot("requested_slot") == TYPE:
-            resolved = resolve_slot_match(
-                tracker.latest_message["text"], type_slot_match
-            )
-            if len(resolved) > 0:
-                return resolved
+            user_input = tracker.latest_message["text"]
 
-        return {}
+            resolved = {}
+            for word in user_input.split(" "):
+                resolved = resolve_slot_match(word, type_slot_match)
+                if len(resolved) > 0:
+                    return resolved
+
+        return ValidateFormFeedback.break_form_if_not_extracted_requested_slot(
```

Comment:
I tried breaking the form if none of the words matched, which it does, but the bot doesn't respond to the user's message :thinking:. Do you see what I am missing?

Reply:
I think it needs to set the requested_slot to None :sweat_smile: 
This is required so the rule to end the form is triggered and finishes properly

Reply:
You can run in interactive mode and see what's the next action

Reply:
I also have found that a good way for an intent to break a form is to specify so in the rules:

https://github.com/RedHatInsights/astro-virtual-assistant/blob/main/data/integrations/rules.yml#L7

This way, it will break forms if the input is not consume (i.e. text slots that capture whatever will still capture this input, we might want to find a way to check if the response has an intent that starts a new flow)

Reply:
> I think it needs to set the requested_slot to None 😅 This is required so the rule to end the form is triggered and finishes properly

It's breaking out of the form fine I think. requested_slot is being set on line 267, It's not responding to the second message, i.e.

```
Your input ->  feedback
? Is your feedback about an error or bug you've experienced? Type out your own message...
Your input ->  i meant openshift
Your input ->
```

Let me try the action_deactivate_loop out :)

Reply:
I'm going to come back to this once you get back, something is confusing me haha.